src/lib/Components/Bidding/Screens/ConfirmBid/index.tsx

```diff
@@ -412,9 +413,11 @@ export class ConfirmBid extends React.Component<ConfirmBidProps, ConfirmBidState
   }
 
   render() {
-    const { artwork, lot_label, sale } = this.props.sale_artwork
+    const { id, artwork, lot_label, sale } = this.props.sale_artwork
     const { requiresPaymentInformation, requiresCheckbox, isLoading } = this.state
     const artworkImage = artwork.image
+    const enablePriceTransparency =
```

Comment:
perhaps `enablePriceSummary` is the most straightforward name (related to a comment above).

Reply:
So, assuming that we'll want to roll out this component to all users in production after beta'ing it with a smaller group of lab users, would we need to have another release of emission and eigen to remove this lab feature switch, therefore rolling it out to everyone?

Just want to make sure that we shouldn't be using Echo here to also enable us to turn this on across all of our devices.

Some relevant docs here that just wanna make sure are surfaced your way: https://github.com/artsy/emission/blob/master/docs/adding_a_lab_option.md

Reply:
> after beta'ing it with a smaller group of lab users

Why do we need that level of feature flagging? Or do we actually need feature flagging at all?